<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmIPAddress</class>
 <widget class="QWidget" name="frmIPAddress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="TumblerDate" name="widgetDate" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="TumblerTime" name="widgetTime" native="true"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnSetTime">
     <property name="text">
      <string>设置时间</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="btnSetDateTime">
     <property name="text">
      <string>设置日期时间</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btnSetDate">
     <property name="text">
      <string>设置日期</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="TumblerDateTime" name="widgetDateTime" native="true"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="ComboBox" name="cbox"/>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="3">
      <widget class="IPAddress" name="widgetIP" native="true"/>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btnSetIP">
       <property name="text">
        <string>设置IP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnGetIP">
       <property name="text">
        <string>获取IP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TumblerDateTime</class>
   <extends>QWidget</extends>
   <header location="global">tumblerdatetime.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
  <customwidget>
   <class>IPAddress</class>
   <extends>QWidget</extends>
   <header>ipaddress.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TumblerDate</class>
   <extends>QWidget</extends>
   <header>tumblerdate.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TumblerTime</class>
   <extends>QWidget</extends>
   <header>tumblertime.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
